src/collection.rs

```diff
@@ -66,6 +66,16 @@ impl Collection
 		}
 	}
 
+	pub fn set_id (&mut self, id: u32)
+	{
+		self.id = id;
+	}
+
+	pub fn id (&self) -> u32
```

Comment:
This should probably be named `get_id`.

Reply:
It shouldn't if we respect the rust api guidelines 

https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

Reply:
ok